tests/functional/api/groups/test_create.py

```diff
@@ -9,145 +9,101 @@
+            {"name": "My Group", "organization": "foobar", "joinable_by": "whoever"},
+            headers=token_auth_header,
+        )
 
         assert res.status_code == 200
 
     def test_it_returns_http_400_with_invalid_payload(self, app, token_auth_header):
-        group = {}
 
         res = app.post_json(
-            "/api/groups", group, headers=token_auth_header, expect_errors=True
+            "/api/groups", {}, headers=token_auth_header, expect_errors=True
         )
 
         assert res.status_code == 400
 
     def test_it_returns_http_400_if_groupid_set_on_default_authority(
         self, app, token_auth_header
     ):
-        group = {"name": "My Group", "groupid": "3434kjkjk"}
         res = app.post_json(
-            "/api/groups", group, headers=token_auth_header, expect_errors=True
+            "/api/groups",
+            {"name": "My Group", "groupid": "3434kjkjk"},
+            headers=token_auth_header,
+            expect_errors=True,
         )
 
         assert res.status_code == 400
 
-    def test_it_returns_http_404_if_no_authenticated_user(
-        self, app, auth_client_header
-    ):
+    def test_it_returns_http_404_if_no_authenticated_user(self, app, auth_header):
         # FIXME: This should return a 403
-        group = {"name": "My Group"}
         res = app.post_json(
-            "/api/groups", group, headers=auth_client_header, expect_errors=True
+            "/api/groups", {"name": "My Group"}, headers=auth_header, expect_errors=True
         )
 
         assert res.status_code == 404
 
     @pytest.mark.xfail
-    def test_it_returns_http_403_if_no_authenticated_user(
-        self, app, auth_client_header
-    ):
-        group = {"name": "My Group"}
+    def test_it_returns_http_403_if_no_authenticated_user(self, app, auth_header):
         res = app.post_json(
-            "/api/groups", group, headers=auth_client_header, expect_errors=True
+            "/api/groups", {"name": "My Group"}, headers=auth_header, expect_errors=True
         )
 
         assert res.status_code == 403
 
-    def test_it_allows_auth_client_with_forwarded_user(
-        self, app, auth_client_header, third_party_user
-    ):
-        headers = auth_client_header
-        headers["X-Forwarded-User"] = third_party_user.userid
-        group = {"name": "My Group"}
+    def test_it_allows_auth_client_with_forwarded_user(self, app, auth_header, user):
+        headers = auth_header
+        headers["X-Forwarded-User"] = user.userid
 
-        res = app.post_json("/api/groups", group, headers=headers)
+        res = app.post_json("/api/groups", {"name": "My Group"}, headers=headers)
 
         assert res.status_code == 200
 
     def test_it_allows_groupdid_from_auth_client_with_forwarded_user(
-        self, app, auth_client_header, third_party_user
+        self, app, auth_headers_3rd_party, user_3rd_party
     ):
-        headers = auth_client_header
-        headers["X-Forwarded-User"] = third_party_user.userid
```

Comment:
Whenever `auth_client_header` referered to the `third_party_user` it was then updated with `X-Forwarded-User`. By making a specific `auth_headers_3rd_party` we can make it easier to understand why these two fixtures are related, and reduce the redundancy of adding this header in every test that calls it.